src/controls/Keyboard.ts

```diff
@@ -43,27 +34,7 @@ function useKeys(keyConfig: KeyConfig[]) {
 }
 
 export function Keyboard() {
-  const { reset, set } = useStore(({ actions: { reset }, set }) => ({ reset, set }))
-  useKeys([
-    { keys: ['ArrowUp', 'w', 'W', 'z', 'Z'], fn: (forward) => set((state) => ({ controls: { ...state.controls, forward } })) },
-    { keys: ['ArrowDown', 's', 'S'], fn: (backward) => set((state) => ({ controls: { ...state.controls, backward } })) },
-    { keys: ['ArrowLeft', 'a', 'A', 'q', 'Q'], fn: (left) => set((state) => ({ controls: { ...state.controls, left } })) },
-    { keys: ['ArrowRight', 'd', 'D'], fn: (right) => set((state) => ({ controls: { ...state.controls, right } })) },
-    { keys: [' '], fn: (brake) => set((state) => ({ controls: { ...state.controls, brake } })) },
-    { keys: ['h', 'H'], fn: (honk) => set((state) => ({ controls: { ...state.controls, honk } })) },
-    { keys: ['Shift'], fn: (boost) => set((state) => ({ controls: { ...state.controls, boost } })) },
-    { keys: ['r', 'R'], fn: reset, up: false },
-    { keys: ['.'], fn: () => set((state) => ({ editor: !state.editor })), up: false },
-    { keys: ['i', 'I'], fn: () => set((state) => ({ help: !state.help, leaderboard: false, pickcolor: false })), up: false },
-    { keys: ['l', 'L'], fn: () => set((state) => ({ help: false, leaderboard: !state.leaderboard, pickcolor: false })), up: false },
-    { keys: ['m', 'M'], fn: () => set((state) => ({ map: !state.map })), up: false },
-    { keys: ['p', 'P'], fn: () => set((state) => ({ help: false, pickcolor: !state.pickcolor, leaderboard: false })), up: false },
-    { keys: ['u', 'U'], fn: () => set((state) => ({ sound: !state.sound })), up: false },
-    {
-      keys: ['c', 'C'],
-      fn: () => set((state) => ({ camera: cameras[(cameras.indexOf(state.camera) + 1) % cameras.length] })),
-      up: false,
-    },
-  ])
+  const [keyboardBindings] = useStore((state) => [state.keyboardBindings])
```

Comment:
```suggestion
  const keyboardBindings = useStore((state) => state.keyboardBindings)
```

Reply:
The problem was in the inappropriate way of how I was creating deduplicated list of keys (traditional javascriptish reference-mistake 😞  ), now I've realised it and fixed, so now it is really no need in array here.